lib/v2/luogu/daily.js

```diff
@@ -9,19 +15,30 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(response.data);
     const title = $('head title').text();
 
-    const firstPost = $('.am-comment-main .am-comment-bd').first();
-    const dailyLink = firstPost.find('a').first().attr('href');
-    const issueHeading = firstPost.find('h1').text().trim();
+    const injection_script = $('head script')
+        .filter((_, el) => $(el).text().startsWith('window._feInjection'))
+        .first()
+        .text();
+    // the window._feConfigVersion and window._tagVersion seems to be a constant, but in cast it will change, write a regexp to match it.
+    const JSONRaw = injection_script.replaceAll('window._feInjection = JSON.parse(decodeURIComponent("', '').replace(/"\)\);window._feConfigVersion=(\d*);window._tagVersion=(\d*);/g, '');
+    const JSONDecode = JSON.parse(decodeURIComponent(JSONRaw));
+
+    const MDRaw = JSONDecode.currentData.post.content;
+    const MDDecode = md.render(MDRaw).replaceAll('\n', '<br />');
```

Comment:
https://github.com/DIYgod/RSSHub/blob/8adc77eb387743057d30e6d544bdae13cd995029/lib/v2/luogu/daily.js#L7
Enabling `breaks` has already [converted](https://github.com/markdown-it/markdown-it?tab=readme-ov-file#init-with-presets-and-options) `\n` into `<br>`